docs/src/pages/components/data-grid/sorting/sorting.md

```diff
@@ -4,34 +4,91 @@ title: Data Grid - Sorting
 
 # Data Grid - Sorting
 
-<p class="description">Sorting allows ordering records in the data grid.</p>
+<p class="description">Easily sort your rows based on one or several criteria.</p>
 
-## Basic sorting
+The sorting rules can be modified through the grid interface by clicking on a column header. Repeat this action to pass to the next sorting direction of the column.
 
-Single column sorting can be triggered with by clicking a column header.
-Repeat this action to change the sorting direction.
+{{"demo": "pages/components/data-grid/sorting/BasicExampleDataGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
-A sorted column can be can pre-configured using the `sortModel` prop:
+## Single and multi-sorting
 
-{{"demo": "pages/components/data-grid/sorting/BasicSortingGrid.js", "bg": "inline"}}
+> ⚠️ The `DataGrid` can only sort the rows according to one criterion at the time.
+>
+> To use multi-sorting, you need to upgrade to the [Pro plan](https://mui.com/store/items/material-ui-pro/)
+
+The demo below let you sort the rows according to several criteria at the same time.
+
+Hold down the <kbd class="key">CTRL</kbd> or <kbd class="key">Shift</kbd> (use <kbd class="key">⌘ Command</kbd> on macOS) key while clicking the column header.
+
+{{"demo": "pages/components/data-grid/sorting/BasicExampleDataGridPro.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Pass sorting rules to the grid
+
+### Structure of the model
+
+The sort model is a list of sorting item.
+Each item represent a sorting rule and is composed of several elements:
+
+- `sortingItem.field`: the field on which we want to apply the rule
+- `sortingItem.sort`: the direction of the sorting (`'asc'`, `'desc'`, `null` or `undefined`, if `null` or `undefined`, the rule will not be applied)
+
+### Initialize the sort model
+
+To initialize the sort model without controlling it, provide the model to the `initialState` prop.
+
+```jsx
+<DataGrid
+  initialState={{
+    sorting: {
+      sortModel: [{ field: 'rating', sort: 'desc' }],
+    },
+  }}
+/>
+```
+
+{{"demo": "pages/components/data-grid/sorting/InitialSort.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Controlled sort model
+
+Use the `sortModel` prop to control the sorting rules applied on the rows.
+
+You can use the `onSortModelChange` prop to listen to changes to the sorting rules and update the prop accordingly.
+
+{{"demo": "pages/components/data-grid/sorting/ControlledSort.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Disable the sorting
```

Comment:
I think this section could be introduced before multi sorting because: It does not need to know the existence of multi-sorting, and it is simpler to understand than sorting model

Reply:
Would you have the same logic for filtering ? 
I kept the same order between the two pages as the logic are similar.

Reply:
I don't know.

Having the same structure is clearly a good point (I was wondering why the flow seems familiar 😅)

I noticed it for this page (and not the filtering one) because here goes from a section with 3 subsections to a small section with one sentence + an example.

Let see if @joserodolfofreitas and @cherniavskii have an opinion

Reply:
Not sure to follow you here

> or remove the configuration altogether

What configuration are we talking about ?

Reply:
You're right, there's no configuration on the `single sorting` and `multi sorting` basic demos (great!). I may have mixed it with another one that sets a default sorting rule (which was the config I was referring to).

Reply:
You can disable the multi sorting on the pro
But I think this option mainly exist to limit on the free, we don't even document it :+1: